src/conn/README.md

```diff
@@ -0,0 +1,32 @@
+# Connection (conn)
+
+## Background Compaction
+
+### Overview
+
+The background compaction thread is implemented in [conn_compact.c](conn_compact.c).
+
+### Compaction Heuristics
+
+They key heuristics used to determine whether a file should be compacted are covered in [__background_compact_should_skip](https://github.com/wiredtiger/wiredtiger/blob/cec2d68c2a51be610745b4ab481c572aaec08d3c/src/conn/conn_compact.c#L233-L311).
+
+These include (order matters):
+1. A table can be configured to be skipped by compaction using the parameter `exclude=["table:a.wt"]`, the configuration is parsed in [__background_compact_exclude_list_process](https://github.com/wiredtiger/wiredtiger/blob/cec2d68c2a51be610745b4ab481c572aaec08d3c/src/conn/conn_compact.c#L100-L120) , you can find an example in [test_compact06](https://github.com/wiredtiger/wiredtiger/blob/cec2d68c2a51be610745b4ab481c572aaec08d3c/test/suite/test_compact06.py#L38)
+1. File size < `1MB`
+1. `background_compact.max_file_skip_time` (unit of Seconds), If the elapsed time since the last compaction run for the same URI does not exceed this threshold, the compaction will be skipped.
```

Comment:
I think we should word this sentence differently:

```suggestion
1. `background_compact.max_file_skip_time` (unit of Seconds). If the elapsed time since the last compaction run for the same URI has exceeded this threshold, the file is selected for compaction.
```